src/lib.rs

```diff
@@ -78,9 +145,9 @@ pub fn parse(contents: &str) -> Result<EditorConfigINIAST, Error<Rule>> {
 
 #[derive(Debug)]
 enum Item {
-	// BlankLine(BlankLine),
-	// Comment(Comment),
-	Property(Property),
+	Raws(Raws),
```

Comment:
I didn't really see a huge difference between `BlankLine` and `Raws` after `Raws` was introduced.